binaries/scheduler/config/local.local

```diff
@@ -12,7 +12,9 @@
     "Type": "stateful",
     "MaxRetriesPerTask" : 2,
     "DebugMode" : false,
-    "RecoverJobsOnStartup" : true
+    "RecoverJobsOnStartup" : true,
+    "DefaultTaskTimeoutMs" : 1800000,
+    "RunnerOverheadMs": 600000
```

Comment:
This seems like a TODO until we can report worker readiness to the scheduler then?